### ISSUE TITLE
Links to rooms in app.element.io open in Tartine

### ISSUE THREAD

**Issue by estellecomment** [tchapgouv/tchap-web-v4#116]
Go to a room in app.element.io
Copy the url
Paste the url in a tchap message
From tartine, click the link you just messaged

Expected : open new tab with app.element.io/<room url>
Obtained : stays in tartine, tries to open the room, fails (because the room number is unknown, probably, makes sense)

Why is tartine opening this link inside tartine, and the browser does not change to element.io ?

**Comment by estellecomment** [comment]
If this is not a security problem, then it's a minor problem (not needed for beta)

We cannot think how this can be used as a security attack : the user that clicks the link will be sent to a room, but cannot join this room if they don't have the permissions (the backend will block this).

**Comment by jdauphant** [comment]
It's not a security issue

**Comment by odelcroi** [comment]
I haven't figured out "why is tartine opening this link inside tartine, and the browser does not change to element.io ?" but I don't think of any security issue. I'll remove the tag beta then and deprioritize the issue

**Comment by giomfo** [comment]
@estellecomment can you confirm the link you copied/pasted has the prefix "https://matrix.to/#"?

**Comment by estellecomment** [comment]
No, it doesn't.

Example : I am on http://< my tchap-v4 server >
I click this link in a message : https://app.element.io/#/room/!< ROOM ID >:matrix.org (not a matrix.to link)
I expect to browse to https://app.element.io/#/room/!< ROOM ID >:matrix.org
Currently in v4, I browse to https://< my tchap-v4 server >/#/room/!< ROOM ID >:matrix.org

It's a potential security problem because the URL that I see in the link is not the URL that I browse to (that's sneaky!)
But I don't see how it can be exploited : I can be directed to the room without my consent if I click an app.element.io link, but that's it. If I don't have access to tchap's < ROOM ID > room, then the room's content is not shown to me.
So I agree with removing security label and deprioritizing. 
It's still a bug in my opinion.

**Comment by giomfo** [comment]
Yes, the origin of the bug should be in the Tchap-web configuration.
I think you kept the Element-web configuration which let the application catch matrix.to links and those which have https://app.element.io/, https://develop.element.io/... prefixes

**Comment by jdauphant** [comment]
> Yes, the origin of the bug should be in the Tchap-web configuration.
> I think you kept the Element-web configuration which let the application catch matrix.to links and those which have https://app.element.io/, https://develop.element.io/... prefixes

Sometime references to element-web are hard coded in the react-sdk (like the feedback module with hard-coded reference to Element-web github).
